plugins/catalog-backend-module-gitlab/src/providers/config.ts

```diff
@@ -35,6 +35,13 @@ function readGitlabConfig(id: string, config: Config): GitlabProviderConfig {
   const projectPattern = new RegExp(
     config.getOptionalString('projectPattern') ?? /[\s\S]*/,
   );
+  const userPattern = new RegExp(
+    config.getOptionalString('userPattern') ?? /[\s\S]*/,
```

Comment:
Probably can update the `config.d.ts` too and add these typings there.

Reply:
done